app/modules/entities/views/work_li.js

```diff
@@ -1,23 +1,36 @@
 import { invertAttr, isOpenedOutside } from 'lib/utils'
 import workLiTemplate from './templates/work_li.hbs'
+import workLiTemplateTable from './templates/work_li_table.hbs'
 import '../scss/work_li.scss'
+import '../scss/work_li_table.scss'
+import { localStorageProxy } from 'lib/local_storage'
 
 export default Marionette.ItemView.extend({
-  template: workLiTemplate,
   className () {
     const prefix = this.model.get('prefix')
     if (this.wrap == null) this.wrap = this.options.wrap
     const wrap = this.wrap ? 'wrapped wrappable' : ''
-    return `workLi entity-prefix-${prefix} ${wrap}`
+    const workLiDisplay = this.display === 'entitiesTable' ? 'workLiTable' : 'workLi'
```

Comment:
bad news: the `initialize` function is called after `className` (thus the convention to always define `className` above, to try to give the hint/reminder), so `this.display` will always be undefined at this point